docs/api/README.md

```diff
@@ -0,0 +1,190 @@
+# Flipper::Api
+
+API for the [Flipper](https://github.com/jnunemaker/flipper) gem.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+    gem 'flipper-api'
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install flipper-api
+
+## Endpoints
+
+### Get all features
+
+`GET /flipper-api/api/v1/features`
```

Comment:
I see what you mean about the mount point. I think I would remove the mount point here and just document `/api/v1/features`. Perhaps a section on how to mount it and how that affects these paths between the installation and endpoints? What do you think?